app/api/api_v1/routers/search.py

```diff
@@ -159,6 +167,69 @@ def download_search_documents(
     )
 
 
+@search_router.post("/searches/download-all-data")
```

Comment:
If we have this as a POST then we won't cache this response at all. I guess we are following this up with other PRs?